Editor/Credentials/Strings.cs

```diff
@@ -136,5 +136,48 @@ public static class Strings
         public const string TabAnywhere = "TabAnywhere";
         public const string TabEC2 = "TabEC2";
         public const string TabHelp = "TabHelp";
+        
+        public const string LabelAccountTitle = "LabelAccountTitle";
+        public const string LabelAccountDescription = "LabelAccountDescription";
+        public const string LabelAccountCardNoAccountTitle = "LabelAccountCardNoAccountTitle";
+        public const string LabelAccountCardNoAccountDescription = "LabelAccountCardNoAccountDescription";
+        public const string LabelAccountCardNoAccountDescriptionLink = "LabelAccountCardNoAccountDescriptionLink";
+        public const string ButtonAccountCardNoAccount = "ButtonAccountCardNoAccount";
+        public const string LabelAccountHasAccountTitle = "LabelAccountHasAccountTitle";
+        public const string LabelAccountHasAccountDescription = "LabelAccountHasAccountDescription";
+        public const string ButtonAccountCardHasAccount = "ButtonAccountCardHasAccount";
+        public const string LabelAccountNewProfileTitle = "LabelAccountNewProfileTitle";
+        public const string LabelAccountNewProfileName = "LabelAccountNewProfileName";
+        public const string LabelAccountNewProfileAccessKey = "LabelAccountNewProfileAccessKey";
+        public const string LabelAccountNewProfileSecretKey = "LabelAccountNewProfileSecretKey";
+        public const string LabelAccountNewProfileRegion = "LabelAccountNewProfileRegion";
+        public const string LabelAccountNewProfileRegionPlaceholder = "LabelAccountNewProfileRegionPlaceholder";
+        public const string ButtonAccountNewProfileCreate = "ButtonAccountNewProfileCreate";
+        public const string ButtonAccountNewProfileCancel = "ButtonAccountNewProfileCancel";
+        public const string LabelAccountNewProfileHelpLink = "LabelAccountNewProfileHelpLink";
+        
+        public const string LabelBootstrapTitle = "LabelBootstrapTitle";
+        public const string LabelBootstrapDescription = "LabelBootstrapDescription";
+        public const string LabelBootstrapPricing = "LabelBootstrapPricing";
+        public const string LabelBootstrapPricingInfo = "LabelBootstrapPricingInfo";
+        public const string LabelBootstrapPricingFreeTier = "LabelBootstrapPricingFreeTier";
+        public const string LabelBootstrapProfileInput = "LabelBootstrapProfileInput";
+        public const string LabelBootstrapBucket = "LabelBootstrapBucket";
+        public const string LabelBootstrapBucketUnset = "LabelBootstrapBucketUnset";
+        public const string LabelBootstrapStatus = "LabelBootstrapStatus";
+        public const string ButtonBootstrapStart = "ButtonBootstrapStart";
+        public const string ButtonBootstrapAnotherProfile = "ButtonBootstrapAnotherProfile";
+        public const string LabelBootstrapHelpLink = "LabelBootstrapHelpLink";
```

Comment:
Refer `TextProvider` for suggestions with variable naming. Any changes there should also be reflected onto this file.

Reply:
Dropped a note in #93, but do you think this file should be moved to a different location perhaps? I see a lot of these strings have nothing to do with `Credentials` but is under the `/Editor/Credentials` directory.

Reply:
As this change isn't to do with the current PR, we will keep note of it and do a new PR that is for file placement.